src/ApolloContext.ts

```diff
@@ -0,0 +1,10 @@
+import createReactContext from 'create-react-context';
+import ApolloClient from 'apollo-client';
+import { DocumentNode } from 'graphql';
+
+export interface ApolloContextValue {
+  client?: ApolloClient<Object>;
+  operations?: Map<string, { query: DocumentNode; variables: any }>;
+}
+
+export default createReactContext<ApolloContextValue | undefined>(undefined);
```

Comment:
Thanks for addressing my previous feedback @hwillson!

Given our concern with bundle sizes (#2743), it seems undesirable to add a new dependency to polyfill `React.createContext`, which has been available unpolyfilled [since React 16.3](https://reactjs.org/blog/2018/03/29/react-v-16-3.html#official-context-api).

If we detect that `React.createContext` is missing here, I think we should print/throw an error recommending either updating to React 16.3+ or manually polyfilling it with `create-react-context`, but we should not do anything that results in adding `create-react-context` to the bundle for developers who are up-to-date with the latest React version.

Reply:
Thanks @benjamn - I was hoping to avoid a major version bump, so the polyfill seemed like a good alternative (given the small size and that we could remove it in `react-apollo` 3). Happy to adjust though - thanks!

Reply:
Is there any way we could fall back to the old behavior if `React.createContext` is not available?

Reply:
I think I found a way to do it. I'll push a commit soon.